raft/src/main/java/org/apache/kafka/raft/ReplicatedCounter.java

```diff
@@ -38,6 +38,9 @@
     private OptionalInt claimedEpoch = OptionalInt.empty();
     private long lastOffsetSnapshotted = -1;
 
+    private int handleSnapshotCalls = 0;
+    private boolean handleSnapshotCalled = false;
```

Comment:
We should only store `handleSnapshotCalls` since `handleSnapshotCalled` is always equal to `handleSnapshotCalls > 0`.

Reply:
fair enough